src/cargo/core/compiler/fingerprint.rs

```diff
@@ -322,6 +323,7 @@ struct DepFingerprint {
     pkg_id: u64,
     name: String,
     public: bool,
+    only_requires_rmeta: bool,
```

Comment:
Can you add a short doc comment explaining this field?  And remove the list above and just doc each field?